README.md

```diff
@@ -16,6 +16,7 @@ You'll need the following dependencies:
 * libgee-0.8-dev
 * libgirepository1.0-dev
 * libgtk-4-dev >= 4.4.0
+* libshumate-dev
```

Comment:
Tiny thing though, maybe we may want to explicit that this is only necessary when building the demo? Because we can build Granite without the demo these days using the `-Ddemo` option.